src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -181,13 +184,15 @@ function MdAutocomplete () {
               ng-if="$mdAutocompleteCtrl.loadingIsVisible()"\
               md-mode="indeterminate"></md-progress-linear>\
           <md-virtual-repeat-container\
+              id="{{menuContainerId}}"\
               md-auto-shrink\
               md-auto-shrink-min="1"\
               ng-mouseenter="$mdAutocompleteCtrl.listEnter()"\
               ng-mouseleave="$mdAutocompleteCtrl.listLeave()"\
               ng-mouseup="$mdAutocompleteCtrl.mouseUp()"\
               ng-hide="$mdAutocompleteCtrl.hidden"\
-              class="md-autocomplete-suggestions-container md-whiteframe-z1"\
+              class="md-autocomplete-suggestions-container md-whiteframe-z1' +
+                (attr.mdMenuContainerClass != null ? ' ' + attr.mdMenuContainerClass : '') + '"\
```

Comment:
Can this use `ng-class` like `menuClass` uses below?

Reply:
Looks like that isn't needed, but the logic here can be pulled out of the template to make things cleaner.